src/app/Http/Controllers/LoginController.php

```diff
@@ -3,11 +3,31 @@
 namespace App\Http\Controllers;
 
 use Illuminate\View\View;
+use Illuminate\Http\Request;
 
 class LoginController extends Controller
 {
     public function index(): View
     {
         return view('auth/login');
     }
+
+    public function login(Request $request)
+    {
+        $request->validate([
+            'loginname' => 'required',
```

Comment:
Can you please moved this in a dedicated Form Request class? 🙇‍♂️

Reply:
+1

Reply:
I already resolved this comment, thank you sir!